packages/utilities/src/dateTimeFormat/absolute.ts

```diff
@@ -24,6 +26,7 @@ export function formatDate(
   options?: Partial<{
     locale: string;
     style: Intl.DateTimeFormatOptions['dateStyle'];
+    timeZone?: Intl.DateTimeFormatOptions['timeZone'];
   }>
```

Comment:
`options` is already wrapped in `Partial<>`, so the `?` after `timeZone` is redundant.